libs/service-portal/access-control/delegations/src/components/access/AccessDate/AccessDate.tsx

```diff
@@ -14,14 +14,15 @@ export const AccessDate = ({ validTo }: AccessDateProps) => {
 
   return (
     <div className={styles.dateContainer}>
-      <Hidden above="sm">
+      {!lg ? (
```

Comment:
Is this the same breakpoint as before?

Reply:
It was not correct as implemented, so I updated it. But good observation 👀